src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -304,29 +308,21 @@ export class PaymentPageView extends View<CartDataType> {
                 }, 2000);
 
                 // Подсвететка не валидных блоков
-                if (!validName) nameInput?.classList.add('invalid');
-                else nameInput?.classList.remove('invalid');
+                if (!valid.validName)setInputValidity(nameInput, valid.validName);
 
-                if (!validAddress) addressInput?.classList.add('invalid');
-                else addressInput?.classList.remove('invalid');
+                if (!valid.validAddress)setInputValidity(addressInput, valid.validAddress);
 
-                if (!validEmail) emailInput?.classList.add('invalid');
-                else emailInput?.classList.remove('invalid');
+                if (!valid.validEmail) setInputValidity(emailInput, valid.validEmail);
 
-                if (!validTell) numberInput?.classList.add('invalid');
-                else numberInput?.classList.remove('invalid');
+                if (!valid.validTell) setInputValidity(numberInput, valid.validTell);
 
-                if (!validCardName) nameCardInput?.classList.add('invalid');
-                else nameCardInput?.classList.remove('invalid');
+                if (!valid.validCardName) setInputValidity(nameCardInput, valid.validCardName);
 
-                if (!validCardNumber) cardNumberInput?.classList.add('invalid');
-                else cardNumberInput?.classList.remove('invalid');
+                if (!valid.validCardNumber) setInputValidity(cardNumberInput, valid.validCardNumber);
 
-                if (!validCardDate) dateInput?.classList.add('invalid');
-                else dateInput?.classList.remove('invalid');
+                if (!valid.validCardDate) setInputValidity(dateInput, valid.validCardDate);
 
-                if (!validCardCvv) cvvInput?.classList.add('invalid');
-                else cvvInput?.classList.remove('invalid');
+                if (!valid.validCardCvv) setInputValidity(cvvInput, valid.validCardCvv);
```

Comment:
Please try to use separate const for it and place it in `utils` file, 
for ex: const highlightInvalidFields = (valid) => {...}